master/buildbot/test/unit/test_changes_p4poller.py

```diff
@@ -334,7 +334,7 @@ def spawnProcess(pp, cmd, argv, env):
                              ['p4', [b'p4', b'login', b'-p']])
             pp.makeConnection(transport)
             self.assertEqual('pass\n', transport.msg)
-            pp.outReceived('Enter password:\nSuccess:  Password verified.\nTICKET_ID_GOES_HERE\n')
+            pp.outReceived('Enter password:\nTICKET_ID_GOES_HERE\n')
```

Comment:
Does this still work with older versions of p4?

Reply:
It is written so that it does (I index relative to the last line for that purpose). That is, both previous and new test case should pass (I didn't bother duplicating the unit test just for that).

But I do not have an older version of p4 to actually try it in production.

Reply:
I think it's worth duplicating the test to ensure that both new and old versions still work.